tests/debug/config.py

```diff
@@ -69,6 +69,7 @@ class DebugConfig(collections.MutableMapping):
         "connect": (),
         # Attach by PID
         "processId": (),
+        "variablePresentation": {},
```

Comment:
This should have the default values for the properties (in case some test-generic code needs to query  what the value is, whether the test had set it or not).